finder-app/manual-linux.sh

```diff
@@ -47,34 +54,68 @@ then
     sudo rm  -rf ${OUTDIR}/rootfs
 fi
 
-# TODO: Create necessary base directories
+mkdir -p ${OUTDIR}/rootfs
+cd ${OUTDIR}/rootfs
+#create root filesystem directories
+mkdir -p bin dev etc home lib lib64 proc sbin sys tmp usr var
+mkdir -p usr/bin usr/lib usr/sbin
+mkdir -p var/log
 
 cd "$OUTDIR"
 if [ ! -d "${OUTDIR}/busybox" ]
 then
 git clone git://busybox.net/busybox.git
     cd busybox
     git checkout ${BUSYBOX_VERSION}
-    # TODO:  Configure busybox
+
+    make distclean
+    make defconfig
 else
     cd busybox
 fi
 
-# TODO: Make and install busybox
+echo "initializing busybox....."
+make ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE}
+make CONFIG_PREFIX=${OUTDIR}/rootfs ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE} install
+
+cd ${OUTDIR}/rootfs
 
 echo "Library dependencies"
 ${CROSS_COMPILE}readelf -a bin/busybox | grep "program interpreter"
 ${CROSS_COMPILE}readelf -a bin/busybox | grep "Shared library"
 
-# TODO: Add library dependencies to rootfs
+echo "Copying required library modules to rootfs"
+# Add library dependencies to rootfs
+cp -r $CUR_DIR/libs/lib .
+cp -r $CUR_DIR/libs/lib64 .
```

Comment:
@maheshchnt rather than add these files to your repo you can just use 

```
export SYSROOT=$(${CROSS_COMPILE}gcc -print-sysroot)
```
to get the location of the compiler sysroot.  Your implementation may break if you cloned this repo on a system with a different version of the cross gcc library installed.  It also relies on adding binary content to the git repo.